### PR TITLE
Android Dessert Migration to Android 14 tag android-14.0.0_r2

### PR DESCRIPTION
**Pull request by swaroopbalan**
Test done : Boot check done

Tracked-On: OAM-112104

### CODE REVIEW

File: include/bsp-celadon.xml [hunk 1]
```diff
@@ -14,23 +14,22 @@
...
   <project name="hardware-intel-bootctrl" path="hardware/intel/bootctrl" remote="github" revision="master" />
   <project name="hardware-intel-usb" path="hardware/intel/usb" remote="github" revision="master" />
   <project name="hdcp" path="hardware/intel/external/media/hdcp" remote="github" revision="master" />
   <project name="kernelflinger" path="hardware/intel/kernelflinger" remote="github" revision="master" />
   <project name="libdmi" path="vendor/intel/libdmi" remote="github" revision="master"/>
-  <project name="libva" path="hardware/intel/external/libva" remote="github" revision="master"/>
   <project name="log_capture" path="vendor/intel/tools/log_capture" remote="github" revision="master"/>
-  <project name="MediaSDK" path="vendor/intel/external/mediasdk_opensource" remote="github" revision="master"/>
+  <project name="MediaSDK" path="vendor/intel/external/mediasdk_opensource" remote="github" revision="master">
+    <annotation name="readonly" value="true"/>
```
Comment (tanujtekriwal) [inline include/bsp-celadon.xml:24]: is readonly needed now?

File: include/bsp-celadon.xml [hunk 2]
```diff
@@ -14,23 +14,22 @@
...
   <project name="MemTrack" path="hardware/intel/display/libmemtrack" remote="github" revision="master"/>
-  <project name="media-driver" path="hardware/intel/external/media/mod" remote="github" revision="master"/>
   <project name="mediasdk-omx" path="vendor/intel/mediasdk_omx_il" remote="github" revision="master"/>
   <project name="mediasdk_release" path="vendor/intel/mediasdk_release" remote="github" revision="master"/>
   <project name="camera-vhal" path="vendor/intel/external/project-celadon/camera-vhal" remote="github" revision="main" />
-  <project name="MediaSDK_C2" path="vendor/intel/mediasdk_c2" remote="github" revision="master"/>
-  <project name="oneVPL" path="vendor/intel/external/onevpl" remote="github" revision="main"/>
-  <project name="oneVPL-Intel-GPU" path="vendor/intel/external/onevpl-intel-gpu" remote="github" revision="main"/>
+  <project name="MediaSDK_C2" path="vendor/intel/mediasdk_c2" remote="github" revision="master">
+    <annotation name="readonly" value="true"/>
```
Comment (tanujtekriwal) [inline include/bsp-celadon.xml:31]: same question

File: include/bsp-celadon.xml [hunk 3]
```diff
@@ -72,4 +65,25 @@
   <project name="linux-intel-lts2021-chromium" path="kernel/lts2021-chromium" remote="github" revision="main"/>
   <project name="linux-intel-lts2021" path="kernel/linux-intel-lts2021" remote="github" revision="main"/>
   <project name="clipboard-agent" path="vendor/intel/external/clipboard-agent" remote="github" revision="main" />
+  <project name="linux-intel-lts2022-chromium" path="kernel/lts2022-chromium" remote="github" revision="main"/>
+  <project name="gmmlib" path="hardware/intel/external/media/gmmlib" remote="github" revision="master">
+    <annotation name="readonly" value="true"/>
```
Comment (tanujtekriwal) [inline include/bsp-celadon.xml:70]: same for the rest below.

**Review by iViggyPrabhu — APPROVED** [review]
Looks Good.

**Pull request merged**